src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__image img {
+    margin: 0 auto;
+    max-width: 160px;
+    display: block;
+    height: 100%;
+  }
+
+  &__text {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--text-color);
+  }
+
+  &__code {
+    margin: 0;
+    padding: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--color);
+  }
+
+  &__review-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 2px;
+    color: var(--text-color);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    margin: 0;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--color);
+  }
+
+  &__price-amount {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: var(--text-color);
+  }
```

Use scss nesting everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 16px;
    
    &-label {
      // write styles here
    }
    
    &-amount {
      // write styles here
    }
  }

```

---

done